[docs]: Update env setup instruction in iosPhysicalDevelopmentBuildLo… (#34871)
expo/expo: docs/scenes/get-started/set-up-your-environment/instructions/iosPhysicalDevelopmentBuildLocal.mdx

### Run the project on your device

> Add the `ios.bundleIdentifier` in the **app.json** file in the root directory to a unique value so that Xcode generates the provisioning profile for app signing step.

Reviewer: ```suggestion
1. Add the `ios.bundleIdentifier` in the **app.json** file in the root directory to a unique value so that Xcode generates the provisioning profile for the app signing step.
```

> Run the following command in your project's root directory and select your plugged in device from the list:

Reviewer: ```suggestion
2. Run the following command in your project's root directory and select your plugged in device from the list:
```

> This command runs a development server after building your app. You can skip running `npx expo start` on the next page.

> > Update `bundleIdentifier` of `app.json` in the root directory to a unique value, or Xcode won’t properly generate the provisioning profile for signing, causing the launch failure.

Reviewer: ```suggestion
> Add the `ios.bundleIdentifier` in the **app.json** file in the root directory to a unique value so that Xcode generates the provisioning profile for app signing.
```

- Suggestion to improve the callout and make it more positive.
- Can you move this callout at the top of this section so that the value is set before the command runs.
